### PR TITLE
feature_2.1

### PR DESCRIPTION
**Pull request by other-dev-profile**

### CODE REVIEW

**Review by glitchy-panda — CHANGES_REQUESTED** [review]
don't like it

**Comment by other-dev-profile** [comment]
> don't like it

why?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # test_2
 
-- feature_2
+- feature_2.1
```
Comment (glitchy-panda) [inline README.md:3]: Don't like it

**Review by glitchy-panda — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (other-dev-profile) [reply README.md:3]: why?

**Comment by other-dev-profile** [comment]
@legan- please review

**Comment by glitchy-panda** [comment]
👍

**Review by glitchy-panda — APPROVED** [review]

**Pull request merged**